### DISCUSSION TITLE
Can you help me with this?

### DISCUSSION THREAD

**Discussion by SKILHUNT** [qwertyo1/tinkoff-trading-bot#4] [Q&A]
Try to use it, but have problem.

[INFO ] 2023-08-09 15:47:18,016 app.strategies.interval.IntervalStrategy: Stop loss triggered. Last price=4.181 figi=BBG004S68473
[ERROR] 2023-08-09 15:47:18,178 tinkoff.invest.logging: 913da3ade29a5444ee7b15be3fc25beb PostOrder INVALID_ARGUMENT 30042
[ERROR] 2023-08-09 15:47:18,178 app.strategies.interval.IntervalStrategy: Failed to post sell order. figi=BBG004S68473. (<StatusCode.INVALID_ARGUMENT: (3, 'invalid argument')>, '30042', Metadata(tracking_id='913da3ade29a5444ee7b15be3fc25beb', ratelimit_limit='300, 300;w=60', ratelimit_remaining=299, ratelimit_reset=45, message='not enough assets for a margin trade'))

Program try sell more, than i have and i cant find why.

**Comment by qwertyo1** [comment]
Hello @SKILHUNT ! Thank you for your message. I believe I won't be able to answer today or tomorrow. But I will reach you this week.

Best regards.

**Comment by qwertyo1** [comment]
I believe it might be related to the feedback I received after the contest. You can find the feedback at https://github.com/qwertyo1/tinkoff-trading-bot/issues/1#issue-1275155516. 
What is the lot size for this FIGI? If it is greater than 1, it could be the bug mentioned in the feedback

**Reply by qwertyo1** [reply]
If it is the case, feel free to propose a pull request with the bug fix. Or I will handle it when have time – this week, I believe.

**Reply by SKILHUNT** [reply]
It really looks like that error with the quantity. I'm bad with programming, if I find a problem, I'll let you know. But I will wait for your decision, it will be much easier for the creator of the code. Thanks for the quick response.

**Reply by qwertyo1** [reply]
Sure, no worries. I'll take a look at it.

**Comment by qwertyo1** [comment]
I've merged a pull request with the fix https://github.com/qwertyo1/tinkoff-trading-bot/pull/5. Please try again. I highly recommend testing in sandbox until you sure that everything works as you expect.

**Comment by SKILHUNT** [comment]
Thanks for your work. I want test on sandbox.

**Comment by SKILHUNT** [comment]
Have another problem on post order.
[Error.docx](https://github.com/qwertyo1/tinkoff-trading-bot/files/12336098/Error.docx)

**Reply by qwertyo1** [reply]
Thanks for reporting it. My bad. Had no chance to test it on holidays since the market was closed :) Will fix it now

**Reply by qwertyo1** [reply]
I've tested it in the sandbox and now there are no errors. Please pull the latest changes. Pull request with the changes: https://github.com/qwertyo1/tinkoff-trading-bot/pull/6
